File: src/main/java/com/indiepost/service/PostScheduledTaskService.java
package com.indiepost.service;

/**
 * Indiepost Scheduled Tasks
 *
 * @deprecated All the scheduled tasks are replaced by AWS Lambda functions.
 */
@Deprecated
public interface PostScheduledTaskService {

    void publishScheduledPosts();

    void rebuildElasticsearchIndices();

    void updateCachedPostStats();
}

File: src/main/kotlin/com/indiepost/service/PostScheduledTaskService.kt
package com.indiepost.service

/**
 * Indiepost Scheduled Tasks
 *
 */
@Deprecated("All the scheduled tasks are replaced by AWS Lambda functions.")
interface PostScheduledTaskService {

    fun publishScheduledPosts()

    fun rebuildElasticsearchIndices()

    fun updateCachedPostStats()
}
